src/__tests__/actions/editArticle/__test__/editArticle.test.js

```diff
@@ -0,0 +1,93 @@
+import thunk from "redux-thunk";
+import configureStore from "redux-mock-store";
+import fetchMock from "fetch-mock";
+import { EditArticle } from "../../../../actions/articleActions/createArticle";
+import { ArticleActionTypes } from "../../../../actions/types";
+
+
+const middleware = [thunk];
+const mockStore = configureStore(middleware);
+
+const { BASE_URL } = process.env;
+
+const data = {
+  title: "test article",
+  description: "this is a test article",
+  body: "Test article by a testuser",
+};
+
+const invalidData = {
+  title: "",
+  description: "",
+  body: "",
+};
+
+const slug = "article-slug";
+
+describe("Edit an article test", () => {
+  beforeEach(() => {
+    fetchMock.restore();
+    localStorage.clear();
+  });
+
+  it("tests update an article", () => {
```

Comment:
Hey @wasibani-roy, Please grammar here, let the tests and test suite descriptions be as descriptive as possible.
Examples of this would be like `it('should update an article')`